src/frontend/js/components/DashboardModerators/ButtonChangeRoleCta/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { messagesDashboardModerators } from '../messages';
+import { Actions, Role } from '../../../types/Enums';
+import { User } from '../../../types/User';
+import { updateUser } from '../../../data/fetchApi';
+
+export interface ButtonModeratorCtaProps {
```

Comment:
```suggestion
export interface ButtonChangeRoleCtaProps {
```

Reply:
now that props are in the same file :-) it would be nicer !